.github/workflows/ci.yml

```diff
@@ -31,7 +31,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-python@v4
         with:
-          python-version: "3.7"
+          python-version: "3.x"
```

Comment:
Also changed this to `3.x`. No need to pin to a concrete version I think.

Reply:
For the CI specifically, we should probably do a matrix of supported versions here: `3.7`, `3.8`, `3.9`, and `3.10`.